ui/src/components/overview/MobileClientCardView.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { Toolbar, Filter, FormControl, EmptyState, EmptyStateTitle, EmptyStateAction, CardGrid } from 'patternfly-react';
+import './MobileClientCardView.css';
+import MobileClientCardViewItem from './MobileClientCardViewItem';
+
+
+const mockServices = [
+  {type: 'metrics'}, {type:'keycloak'}, {type: 'sync'}, {type: 'push'}
+];
+
+const mockBuilds = [
+  {status: "success"}, {status: "failed"}, {status: "success"}, {status: "failed"}
+];
+
+class MobileClientCardView extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {filter: "", currentValue: ""};
+  }
+
+  setFilter(filterValue) {
+    this.setState({filter: filterValue});
+  }
+
+  updateCurrentValue(event) {
+    this.setState({ currentValue: event.target.value });
+  }
+
+  onValueKeyPress(keyEvent) {
+    const { currentValue } = this.state;
+    if (keyEvent.key === 'Enter' && currentValue && currentValue.length > 0) {
```

Comment:
shouldn't be there a way to do the filtering without keyboard?

Reply:
how?